packages/paste-core/components/anchor/src/index.tsx

```diff
@@ -22,21 +24,30 @@ export const secureExternalLink = (href: string): {} | undefined => {
   };
 };
 
-const Anchor = React.forwardRef<HTMLAnchorElement, AnchorProps>(({variant = 'default', ...props}, ref) => {
-  const AnchorComponent = AnchorVariants[variant];
-
-  return (
-    <AnchorComponent
-      href={props.href}
-      ref={ref}
-      variant={variant}
-      {...secureExternalLink(props.href)}
-      {...safelySpreadTextProps(props)}
-    >
-      {props.children}
-    </AnchorComponent>
-  );
-});
+const Anchor = React.forwardRef<HTMLAnchorElement, AnchorProps>(
+  ({variant = 'default', showExternal, ...props}, ref) => {
+    const AnchorComponent = AnchorVariants[variant];
+
+    return (
+      <AnchorComponent
+        href={props.href}
+        ref={ref}
+        variant={variant}
+        {...secureExternalLink(props.href)}
+        {...safelySpreadTextProps(props)}
+      >
+        {showExternal ? (
+          <Box as="span" display="inline-flex" alignItems="center">
+            {props.children}
+            <LinkExternalIcon decorative={false} title="link takes you to an external page" />
```

Comment:
Super nit. Maybe:
```suggestion
            <LinkExternalIcon decorative={false} title="link takes you to an external URL" />
```

Reply:
So I think from a user perspective, page is a more common and approachable term